src/test/groovy/org/mockito/release/internal/gradle/util/DefaultReleaseNotesSerializerTest.groovy

```diff
@@ -0,0 +1,39 @@
+package org.mockito.release.internal.gradle.util
+
+import org.apache.commons.lang.builder.EqualsBuilder
```

Comment:
If you want to use commons lang, please add explicit testCompile dependency. You're getting commons lang transitively via Gradle API :) Explicit dependencies improve transparency and are safer in case Gradle API stops shipping / isolates its dependencies.

I'm ok to add more dependencies for testing. However, we should avoid adding compile dependencies because they make the library bigger and increase liability.